File: Exemplos/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Java-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/TERMICA/ImprimeXMLNFCe.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA

import com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA.TermicaCommand

class ImprimeXMLNFCe(
    private val dados: String,
    private val indexcsc: Int,
    private val csc: String,
    private val param: Int
) : TermicaCommand("ImprimeXMLNFCe") {
    override fun functionParameters(): String {
        return "\"dados\"" + ":" + "\"" + dados + "\"" + "," +
                "\"indexcsc\"" + ":" + indexcsc + "," +
                "\"csc\"" + ":" + "\"" + csc + "\"" + "," +
                "\"param\"" + ":" + param
    }
}

File: Exemplos M10/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/TERMICA/imprime_xml_nfce.dart
import 'package:intent_digital_hub/IntentDigitalHubService/TERMICA/termica_commad.dart';

class ImprimeXMLNFCe extends TermicaCommand {
  final String _dados;
  final int _indexcsc;
  final String _csc;
  final int _param;

  ImprimeXMLNFCe(this._dados, this._indexcsc, this._csc, this._param)
      : super('ImprimeXMLNFCe');

  @override
  functionParameters() {
    return '"dados"'
        ':'
        '"'
        '$_dados'
        '"'
        ','
        '"indexcsc"'
        ':'
        '$_indexcsc'
        ','
        '"csc"'
        ':'
        '"$_csc"'
        ','
        '"param"'
        ':'
        '$_param';
  }
}
